diagnostic_updater/include/diagnostic_updater/diagnostic_updater.hpp

```diff
@@ -560,7 +560,7 @@ class Updater : public DiagnosticTaskVector
       node_->create_publisher<diagnostic_msgs::msg::DiagnosticArray>(
       "/diagnostics", 1);
 
-    period_ = static_cast<rcl_duration_value_t>(1.0);
+    period_ = static_cast<rcl_duration_value_t>(1000000000);  // 1.0s
```

Comment:
what about changing this to `1e9` for readability?

Reply:
Sure!

Reply:
Done.